sdk/monitor/monitor-opentelemetry/src/browserSdkLoader/snippetInjectionHelper.ts

```diff
@@ -0,0 +1,150 @@
+import * as zlib from "zlib";
```

Comment:
Let use browser SDK loader term everywhere, including our classes and configuration names to be consistent

Reply:
My thought process here was that these helper functions are specifically designed to help with injecting the snippet as a part of loading the browser sdk, but I can unify these with the naming convention since I can see how that'd be confusing to use both terms.